### ISSUE TITLE
Ability To Hide White Cards While During Round

### ISSUE THREAD

**Issue by JudsonStevens** [jakelauer/allbadcards#24]
Part of the fun of playing in person is listening to the person read out the cards - it would be awesome if there was the ability to hide white cards from other players during the round. This could be switched on and off at will.

**Comment by jakelauer** [comment]
I love this idea.

**Comment by JudsonStevens** [comment]
Awesome! We ended up just simulating it by having person read the black card and then hitting next to display the white card at the point in the sentence it was supposed to be. That actually made for some fun as the black card holder didn't know what was going to be there. So honestly, either way - it would be nice to have the ability though.

I took a look at the code but I don't know enough TypeScript to have anything useful right off the bat. Are you open to submissions at some point? For this, or something else later on if you knock this out.
